src/main/java/qna/domain/Answer.java

```diff
@@ -3,15 +3,30 @@
 import qna.NotFoundException;
 import qna.UnAuthorizedException;
 
+import javax.persistence.*;
 import java.util.Objects;
 
-public class Answer {
+@Entity
+@Table(name = "answer")
+public class Answer extends BaseEntity{
```

Comment:
```suggestion
public class Answer extends BaseEntity {
```

중요한 부분은 아니지만 코드 정렬을 한번 해주시면 좋을 것 같네요!

Reply:
혹시 부분 정렬인지 알 수 있을까요?
BaseEntity { 
이구간을 말씀해 주시는걸까요?

Reply:
네 맞습니다.

IDE의 정렬 기능을 사용하면 위와 같은 부분이 자동 정렬될 것 같네요.

중요한 부분은 아니지만 기본적인 부분이라 코멘트 남겨드렸습니다.

Reply:
2단계에서는 정렬하고 진행하겠습니다. 감사합니다!